.github/issue_template.md

```diff
@@ -0,0 +1,10 @@
+Issue overview
+--------------
+Please change this line to a description of the defect, with useful supporting information including how the issue was found, conditions that trigger this defect, workarounds for users of the current version, and a clear description of how to validate that a fix is complete for this defect.
+
+### Checklist
+Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
```

Comment:
Probably need to add version, operating system to the list of information. We might not get too many issues from users of the various GUIs, but it might be good to have something about there here as well.